src/GraphRequest.ts

```diff
@@ -501,19 +494,29 @@ export class GraphRequest {
 	 * @returns The same GraphRequest instance that is being called with
```

Comment:
can you comment here on what the modification the graphrequest is.  It currently reads like I passed in something and got the same thing back.

Reply:
Added more information in some comments. There are more comments like this one. Will change them in another PR.